Makefile

```diff
@@ -36,6 +38,10 @@ verify: ## Verify code. Includes dependencies, linting, formatting, etc
 	go fmt ./...
 	golangci-lint run
 
+verify-licenses: ## Verifies dependency licenses and requires GITHUB_TOKEN to be set
+	go build $(GOFLAGS) -o $(BUILD_DIR)/karpenter cmd/controller/main.go 
```

Comment:
Consider just dropping the binary at root and adding a .gitignore

Reply:
We'll be adding some more files that will need to be temporarily held and gitignored. I generally like the build dir that acts as a temp dir for the repo.  Any specific reasons you're against that approach?

Reply:
The generated release assets like helm chart, docs, and k8s yamls.  I suppose they don't have to be ignored if we do the PR and generation in one step.

Reply:
removed build dir